### ISSUE TITLE
Move from Fork to Clone

### ISSUE THREAD

**Issue by VikashKotharyWork** [ankicommunity/anki-sync-server#2]
- Forks are treated as second class citizens on GitHub.

**Comment by Sterrs** [comment]
Last time I did this, I had to contact GitHub directly in order to request this

**Comment by VikashKothary** [comment]
Yep. The only reason we've kept the 'fork' status is because I believe it may help people find this project. But maybe there is a better way to do this.

**Comment by reivilibre** [comment]
I have been in confusion about which anki sync server is 'the' right one to use; being a fork definitely makes it feel like this is not the one, especially as it is a fork of dsnopek's when tsudoko's is more modern than that one.

The readme also says

> This version is a fork of jdoe0/ankisyncd. It supports Python 3 and Anki **2.1**

which adds further to the confusion.

I would agree with the idea of making this a non-fork and removing the above wording (or enhancing it so it is clear that this should be the *one true repository* going forward).

**Comment by AntonOfTheWoods** [comment]
I'll put it here as well - I would completely remove all history before our first commits and put the commit we started with in the readme with a link to that commit in the original. At the top of the readme should be a short explanation of what we did and why we did it. 

We could also adopt a "latest compatible tested desktop client" versioning - so if we have validated with 2.1.35, then the server becomes 2.1.35. Previously I guess it was the git submodule, so it ended up being that.

What do you think?

**Comment by VikashKothary** [comment]
Hi @reivilibre,

You make extremely valid points. Let's pick up this ticket and get it resolved. You've also made some other valid points about clarity and discovery which we'll need to action before we can close this ticket.

@AntonOfTheWoods Like I mentioned in the other ticket, deleting the commit history is quite controversial. As such, I've created a separate ticket for it. Regarding, the versioning, I've created a separate ticket for that conversation. See: #42.

Here's what I think the actions are:

- [ ] **Make this repo a stand-alone**: This involves an email to Github by a repo admin. I can take this action. 
- [ ] **Update README**: We should include a list of all the different forks that were used in this repo. I've added a comment to #3 .
- [ ] **Release v2.2.0**: That's a nice catch. No idea how this happened but should be a quick fix. I've created a new issue: #42.
- [ ] **Fix the add-on issue**: Could you add a comment on #34 on the issue you had and the resolution. The addon is about to become its own project and we wouldn't want to export it with issues that are only documented in this project.
- [ ] (optional) **Update tsudoko's README**: This would be ideal but I've not heard back from tsudoko in a while. So we'll see.

Anything I missed? 
Hopefully making these changes will improve the clarity and discovery for the project.

**Comment by VikashKothary** [comment]
I'm going to consider this ticket resolved. There are now newer tickets which handle all the points above.
